src/printer.js

```diff
@@ -3002,7 +3002,7 @@ function printNode(path, options, print) {
     case "inline":
       return join(
         literalline,
-        node.raw.replace("___PSEUDO_INLINE_PLACEHOLDER___", "").split(/\r?\n/g)
+        node.raw.replace("___PSEUDO_INLINE_PLACEHOLDER___", "").split("\n")
       );
```

Comment:
FYI: There is a `doc.utils.replaceEndOfLine` we can use, but I'm not sure if it's available in supported Prettier versions, so I'm not going to touch it.